controllers/datadogagent/feature/externalmetrics/rbac.go

```diff
@@ -0,0 +1,146 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package externalmetrics
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/common"
+	"github.com/DataDog/datadog-operator/pkg/kubernetes/rbac"
+)
+
+func getDCAClusterPolicyRules(useDDM, wpaController bool) []rbacv1.PolicyRule {
```

Comment:
would it make sense if for the latter arg we used similar syntax such as `useWPA` ?